### PR TITLE
Hover card jump fix

### LINKED ISSUE #9132: Hover card jump
### Environment Information 
 
- __Package version(s)__: (fill this out) 
- __Browser and OS versions__: (fill this out if relevant)

#### Actual behavior:
 
The hover card is "jumping" if user move mouse into the card when the card is rendering
fabric_hovercard_jump

Reproable on fabric display site: https://developer.microsoft.com/en-us/fabric#/controls/web/hovercard 
 
#### Expected behavior:
 
The card should not "jump" when it is rendering

#### Priorities and help requested:
 
Are you willing to submit a PR to fix? (Yes, No)
 
Requested priority: (Blocking, High, Normal, Low) 
 
Products/sites affected: (if applicable)

### PR DESCRIPTION
**Pull request by sopranopillow**
#### Pull request checklist

- [x] Addresses an existing issue: Fixes #9132
- [x] Include a change request file using `$ npm run change`

#### Description of changes

This PR fixes issue #9132 in which when expanding the hovercard, it jumps up and down depending on the space around it. 

#### Focus areas to test

(optional)

###### Microsoft Reviewers: [Open in CodeFlow](http://wpcp.azurewebsites.net/CodeFlowProtocolProxy2.php?pullrequest=https://github.com/OfficeDev/office-ui-fabric-react/pull/9338)

### CODE REVIEW

File: packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx [hunk 1]
```diff
@@ -76,12 +76,13 @@ export class ExpandingCardBase extends BaseComponent<IExpandingCardProps, IExpan
   private _onRenderExpandedCard = (): JSX.Element => {
     // firstFrameRendered helps in initially setting height of expanded card to 1px, even if
     // mode prop is set to ExpandingCardMode.expanded on first render. This is to make sure transition animation takes place.
-    !this.state.firstFrameRendered &&
```
Comment (Vitalius1) [inline packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx]: @atneik you were the one who worked initially on this implementation and I know this line was important on making the animation work correctly.

**Comment by Vitalius1** [comment]
@sopranopillow , @JasonGore I can't reproduce this on our website... 🤔 Is it a specific browser?

**Comment by sopranopillow** [comment]
I was able to repro it in edge, chrome and firefox. If there's not enough space under the link for the hovercard to display, it shows the jump most of the tries.

File: packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx (hunk 1 above)
Comment (atneik) [reply packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx]: With this change, we are setting `firstFrameRendered` state as `true` async, even if it was already `true`. I'm trying to understand how is this fixing the issue?

Comment (atneik) [reply packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx]: Moreover, this is causing an infinite loop as long as the card is open.

Comment (JasonGore) [reply packages/office-ui-fabric-react/src/components/HoverCard/ExpandingCard.base.tsx]: I have been able to repro the issue pretty consistently based on window size. I think it's something that should be fixed. Do you know of a bitter fix @atneik ?

**Comment by dzearing** [comment]
@sopranopillow @atneik @Vitalius1 Let's not this go stale. Can you guys teams chat about this tomorrow and get this resolved?

**Comment by atneik** [comment]
@dzearing, @sopranopillow is on it

**Pull request closed without merging**